JAVA/currency-converter/README.md

```diff
@@ -0,0 +1,72 @@
+
+# Currency Converter
+
+<p align="center">
+<img src="blob/currency-converter-screenshot.png">
+</p>
+
+It is the GUI Currency Converter written in Java with the help of JavaFX. In currency converter we can interconvert currencies.
+
+## **Quick Start**
+- Clone this repository
+
+``` 
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+```
+- Switch Branch
+
+```
+git checkout currency-converter
+```
```

Comment:
i think so switching branch will be not there in the DevScript Repo because thay will not found any branch on this name so remove this then guide the user how to reach at your project after cloning  like cd... to this folder, then install the necessary packages which are required with cli commands after that add final command to run the project in local.